client/src/components/commonComponents/Dropdown/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './index.css';
+
+const langs = [
+  {
+    id: 1,
+    language: 'Chinese',
+  },
+  {
+    id: 2,
+    language: 'Spanish',
+  },
+  {
+    id: 3,
+    language: 'English',
+  },
+  {
+    id: 4,
+    language: 'Hindi',
+  },
+  {
+    id: 5,
+    language: 'Russian',
+  },
+];
+
+const Dropdown = ({ label, name, languages = langs }) => {
```

Comment:
```suggestion
const Dropdown = ({ label, name, languages  }) => {
```
remove the langs and in the PR description you can write how to use your component

Reply:
Okay removed the default value and will explain in the pr how to use